src/ExtInterface/ReefMod/ReefModDomain.jl

```diff
@@ -209,7 +209,7 @@ function load_initial_cover(
     # as suggested by YM (pers comm. 2023-08-08 12:55pm AEST). Distribution is used to split ReefMod initial
     # species covers into ADRIA's 6 size classes by weighting with the cdf.
     reef_mod_area_dist = LogNormal(log(700), log(4))
-    bin_edges_area = colony_mean_area(bin_edges())
+    bin_edges_area = colony_mean_area(bin_edges(; unit=:cm))
```

Comment:
Is this being consistently used for other models too?

Reply:
CoralBlox uses `bin_edges()` correctly.